src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -81,4 +81,6 @@ public net.minecraft.world.level.biome.Biome m_47505_(Lnet/minecraft/core/BlockP
 protected net.minecraft.world.level.block.entity.BlockEntity f_58856_ # blockState
 public net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase f_60599_ # destroySpeed
 public net.minecraft.world.level.block.state.BlockBehaviour$Properties f_60886_ # lightEmission
-public net.minecraft.world.level.levelgen.placement.PlacedFeature m_191797_(Lnet/minecraft/world/level/levelgen/placement/PlacementContext;Ljava/util/Random;Lnet/minecraft/core/BlockPos;)Z # placeWithContext
\ No newline at end of file
+public net.minecraft.world.level.levelgen.placement.PlacedFeature m_191797_(Lnet/minecraft/world/level/levelgen/placement/PlacementContext;Ljava/util/Random;Lnet/minecraft/core/BlockPos;)Z # placeWithContext
+
+public net.minecraft.client.renderer.LevelRenderer m_109637_(Lcom/mojang/blaze3d/vertex/PoseStack;Lcom/mojang/blaze3d/vertex/VertexConsumer;Lnet/minecraft/world/entity/Entity;DDDLnet/minecraft/core/BlockPos;Lnet/minecraft/world/level/block/state/BlockState;)V # renderHitOutline
```

Comment:
This should be moved higher up in the AT. For the most part similar to forge and their own AT we try and keep this file sorted alphabetically by package class method/field